client/views/windows/sign_in.tpl

```diff
@@ -2,15 +2,16 @@
 	<img src="img/logo-vertical-transparent-bg.svg" class="logo" alt="The Lounge" width="256" height="170">
 	<img src="img/logo-vertical-transparent-bg-inverted.svg" class="logo-inverted" alt="The Lounge" width="256" height="170">
 
-	<label>
-		Username
-		<input class="input" name="user" autofocus>
-	</label>
+	<label>Username</label>
+	<input class="input" name="user" autofocus>
 
-	<label>
-		Password
+	<div class="password-container">
+		<label>Password</label>
 		<input class="input" type="password" name="password">
-	</label>
+		<span class="reveal-password tooltipped tooltipped-n tooltipped-no-delay" aria-label="Show Password" data-alt-label="Hide Password">
+			<i></i>
```

Comment:
I don't think we need (should?) have an `i` elements for those. If you look at other icon buttons, you'll see we don't wrap those in `i` elements. That will probably mean you'll need a container for the tooltip. That's rather unfortunate, but that's consistent with the rest so that when we improve/change things, we can do it all over the place more easily.